connectors/resteasy/pom.xml

```diff
@@ -19,6 +19,12 @@
 			<groupId>org.jboss.resteasy</groupId>
 			<artifactId>resteasy-jaxrs</artifactId>
 			<version>${resteasy-version}</version>
+			<exclusions>
+				<exclusion>
+					<artifactId>httpclient</artifactId>
+					<groupId>org.apache.httpcomponents</groupId>
+				</exclusion>
+			</exclusions>
```

Comment:
We tried using higher version of resteasy jar. But it did not work because of the version resteasy usage of httpclient. There are few SSL related bug fix only in 4.5.3 version.
https://archive.apache.org/dist/httpcomponents/httpclient/RELEASE_NOTES-4.5.x.txt